index.html

```diff
@@ -34,10 +34,11 @@ <h2 class="tel"><a href="tel:+380990671174">+38(099) 067-11-74</a></h2>
         </div>
     </header>
     <main>
-        <div class="work">
-            <h3 class="text-about-us">
-                Замовте майстра на всі руки, якому можна довіряти</h3>
-        </div>
+        <div class="hero-section">
+            <h3 class="motto">
+                <br>Замовте майстра на всі руки, якому можна довіряти! 
```

Comment:
Also, if you want to add some offset from top, please use padding or margin instead of `<br>`. Perhaps, you wanted to center this motto vertically? In this case, use flexbox tools to achieve it.